### ISSUE TITLE
Upgrading issues: client/styles/application.scss.css: missing '}' near line 186:3

### ISSUE THREAD

**Issue by cbelsole** [englue/meteor-bootstrap3-sass#27]
I recently upgraded this package from 3.3.4_4 to 3.3.5_1 and now I am getting 
```
While building the application:
   client/styles/application.scss.css: missing '}' near line 186:3
```
I removed all of the scss from the app except what was in this package. So I am pretty sure it is coming from here.

**Comment by reywood** [comment]
It's likely that your scss package is out of date. Which one are you using? If it's `stevezhu:sass`, I noticed that it's using an older version of the node-sass npm package. I've opened an issue (stevezhu/meteor-sass#8). If you switch to the `fourseven:scss` package, it should work.

**Comment by dannyvassallo** [comment]
@cbelsole @reywood I get the same error ```missing '}' near line 186:3``` and this occurs whether I'm using ```stevezhu:sass``` or ```fourseven:scss```.

**Comment by reywood** [comment]
Please post the contents of your `.meteor/versions` file.

**Comment by dannyvassallo** [comment]
I needed to deploy to heroku and this was an issue standing in the way so I've temporarily switched to less to avoid the errors. This is a version file from a previous commit. After some digging I found it may be the fourseven version I was using so I specified version 2.1.1. The error persisted regardless of the SASS version I was running.

```
accounts-base@1.2.0
accounts-password@1.1.1
accounts-ui@1.1.5
accounts-ui-unstyled@1.1.7
artwells:accounts-guest@0.1.12
autopublish@1.0.3
autoupdate@1.2.1
base64@1.0.3
binary-heap@1.0.3
blaze@2.1.2
blaze-tools@1.0.3
boilerplate-generator@1.0.3
callback-hook@1.0.3
check@1.0.5
coffeescript@1.0.6
ddp@1.1.0
deps@1.0.7
ejson@1.0.6
email@1.0.6
fastclick@1.0.3
fortawesome:fontawesome@4.3.0
fourseven:scss@2.1.1
geojson-utils@1.0.3
harrison:papa-parse@1.1.0
html-tools@1.0.4
htmljs@1.0.4
http@1.1.0
id-map@1.0.3
iron:controller@1.0.8
iron:core@1.0.8
iron:dynamic-template@1.0.8
iron:layout@1.0.8
iron:location@1.0.9
iron:middleware-stack@1.0.9
iron:router@1.0.9
iron:url@1.0.9
jquery@1.11.3_2
json@1.0.3
launch-screen@1.0.2
less@1.0.14
livedata@1.0.13
localstorage@1.0.3
logging@1.0.7
meteor@1.1.6
meteor-platform@1.2.2
minifiers@1.1.5
minimongo@1.0.8
mizzao:animated-each@0.2.0
mizzao:bootboxjs@4.4.0
mizzao:build-fetcher@0.2.0
mobile-status-bar@1.0.3
mongo@1.1.0
nemo64:bootstrap@3.3.5_2
nemo64:bootstrap-data@3.3.5
npm-bcrypt@0.7.8_2
observe-sequence@1.0.6
ordered-dict@1.0.3
random@1.0.3
reactive-dict@1.1.0
reactive-var@1.0.5
reload@1.1.3
retry@1.0.3
routepolicy@1.0.5
service-configuration@1.0.4
session@1.1.0
sha@1.0.3
spacebars@1.0.6
spacebars-compiler@1.0.6
srp@1.0.3
templating@1.1.1
theduke:bootstrap-modal-prompt@0.0.2
tracker@1.0.7
ui@1.0.6
underscore@1.0.3
url@1.0.4
webapp@1.2.0
webapp-hashing@1.0.3
zeroasterisk:throttle@0.3.2
```

**Comment by reywood** [comment]
Version 2.1.1 of `fourseven:scss` does not support some SCSS language features used in bootstrap-sass 3.3.5. Version 3.2.0 of `fourseven:scss` (which was available when version 3.3.5_1 of `reywood:bootstrap3-sass` was released) does and will successfully compile this package. Note that if you had a 2.x.x version of `fourseven:scss` installed, `meteor update` will not upgrade your project to a 3.x.x version. You have to remove and re-add the package.

```sh
$ meteor remove fourseven:scss
$ meteor add fourseven:scss
```
